applications/portal/frontend/src/components/home/ExperimentList.jsx

```diff
@@ -227,7 +306,7 @@ const ExperimentList = (props) => {
             label={'All tags'}
           />
           {
-            tags.map((tag, i) => <FormControlLabel
+            tagsOptions.map((tag, i) => <FormControlLabel
```

Comment:
If there are no tags we probably should provide that information to the user somehow (this is just a nice to have)